### PR TITLE
chore(ckbtc): proposals to install Bitcoin Checker and to use it in ckBTC minter

### PR DESCRIPTION
**Pull request by ninegua**

### CODE REVIEW

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Previous ckBTC minter proposal: https://dashboard.internetcomputer.org/proposal/134172
+
+---
+
+## Motivation
+
+Upgrade the ckBTC minter canister to use the BTC Checker canister instead of the KYT canister.
+
+The BTC Checker canister (`oltsj-fqaaa-aaaar-qal5q-cai`) is a new canister that is installed by proposal XXXXX.
```
Comment (ninegua) [inline rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md:21]: TODO: replace `XXXXX` with the actual proposal number and link.

File: rs/bitcoin/ckbtc/mainnet/btc_checker_install_2024_12_06.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+---
+
+## Motivation
+
+Install the Bitcoin Checker canister at canister id `oltsj-fqaaa-aaaar-qal5q-cai`.
+
+The Bitcoin Checker implements checks for Bitcoin transactions and addresses against the OFAC (https://sanctionssearch.ofac.treas.gov/) list.
+It offers a more transparent approach than the KYT canister, and helps to simplify the process.
+A followup NNS proposal will propose to switch The ckBTC minter to using the Bitcoin Checker.
```
Comment (gregorydemay) [inline rs/bitcoin/ckbtc/mainnet/btc_checker_install_2024_12_06.md:21]: nit:
```suggestion
A follow-up NNS proposal will propose to switch the ckBTC minter to using the Bitcoin Checker.
```

File: rs/bitcoin/ckbtc/mainnet/btc_checker_install_2024_12_06.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Proposal to install the BTC Checker canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `9849a2f03af855d09ac42f5949393c86df3d9c47`
+
+New compressed Wasm hash: `455f2e7a2a1b872b59c7dc6729f77f507ab4c4617be402d791c9e79ec55bf6d4`
+
+Install args hash: `4bf28e4112db80ce139abc509a4178a6790582566a5c64c5fe2ad009b4173133`
```
Comment (gregorydemay) [inline rs/bitcoin/ckbtc/mainnet/btc_checker_install_2024_12_06.md:9]: confirmed 🎉

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+# Proposal to upgrade the ckBTC minter canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `9849a2f03af855d09ac42f5949393c86df3d9c47`
+
+New compressed Wasm hash: `ba096af5f4c4459e6f4ba7c3e663f2287aec9ae359fc94faa35f87e6a4114a28`
```
Comment (gregorydemay) [inline rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md:7]: confirmed 🎉

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+Upgrade args hash: `90fcd3ef7d62d0eb77d520dbdc0fe923bee5061930de9299e579bc080fd0a858`
```
Comment (gregorydemay) [inline rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md:9]: confirmed 🎉

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+---
+
+## Motivation
+
+Upgrade the ckBTC minter canister to use the BTC Checker canister instead of the KYT canister.
+
+The BTC Checker canister (`oltsj-fqaaa-aaaar-qal5q-cai`) is a new canister that is installed by proposal XXXXX.
+It implements checks for Bitcoin transactions and addresses against the OFAC (https://sanctionssearch.ofac.treas.gov/) list.
+BTC to ckBTC deposits that had previously quarantined UTXOs can be retried by the user.
+Such deposits will be re-evaluated under the new approach.
```
Comment (gregorydemay) [inline rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_12_06.md:24]: nit: slight reformulation
```suggestion
BTC to ckBTC deposits that had previously quarantined or ignored UTXOs can be retried by the user.
Such deposits will be re-evaluated under the new approach.
```

**Review by gregorydemay — APPROVED** [review]
Thanks @ninegua for making the proposals!

**Pull request merged**